keyboards/minidox/keymaps/pvinis/keymap.c

```diff
@@ -0,0 +1,181 @@
+// pvinis minidox
+// ,------------------------.          ,------------------------.
+// |    |    |    |    |    |          |    |    |    |    |    |
+// |----+----+----x----x----|          |----x----x----+----+----|
+// |    |    |    |    |    |          |    |    |    |    |    |
+// |----+----+----x----x----|          |----x----x----+----+----|
+// |    |    |    |    |    |          |    |    |    |    |    |
+// `------------------------'          `------------------------'
+//             ,--------------.      ,--------------.
+//             |    |    |    |      |    |    |    |
+//             `---------|    |      |    |----+----.
+//                       |    |      |    |
+//                       `----'      `----'
+
+
+#include QMK_KEYBOARD_H
+#include "pvinis.h"
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// Fillers to make layering more clear
```

Comment:
Lines 36-38 should be deleted. QMK defines these keycode aliases automatically.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31